src/mon-user/mon-user.module.ts

```diff
@@ -0,0 +1,14 @@
+import { Module } from '@nestjs/common';
+import { MongooseModule } from '@nestjs/mongoose';
+import { MonUsersController } from './mon-user.controller';
+import { MonUser, UserSchema } from './schemas/mon-user.schema';
+import { MonUserService } from './mon-user.service';
+
+@Module({
+  imports: [
+    MongooseModule.forFeature([{ name: MonUser.name, schema: UserSchema }]),
+  ],
+  controllers: [MonUsersController],
```

Comment:
In 6th homework we had some workwith the abstractions, I think it will be interesting for you
because you are a skillfull one ;)